apps/web/tsconfig.json

```diff
@@ -17,7 +17,7 @@
     "**/*",
     "**/*.json",
     "../../types/**/*"
-  ],
```

Comment:
This seems suspicious to me, none of the other base-ui files needed it, why would this?

Reply:
removed.